scripts/post_npm_install.mjs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The Outline Authors
```

Comment:
Let's put this file under `third_party/jsign`, so we can properly keep track of third-party dependencies. Add the LICENSE file there. See go/releasing/preparing#third-party-components and https://goto.google.com/thirdparty/non-google3

Reply:
Then we don't need to download it during build, we can simply put it under third_party and check it in.

Reply:
That's an option too, which makes the build a bit more hermetic. I guess the tradeoff is that it makes it a bit harder to update and it makes the repository larger.

An another approach, which makes dependencies a bit more explicit, is to create a `third_party/jsign/package.json` that fetches the file on install and declares the license field. Then the client can depend on it as an explicit dev dependency. You can use folders as dependencies: https://docs.npmjs.com/about-packages-and-modules#about-package-formats. This provides a nice location for the downloaded binary under node_modules.

This can be a nice way to encapsulate all that is needed for that dependency.

Reply:
I like the later idea. Will try to implement it this way.